.github/workflows/release.yml

```diff
@@ -21,6 +21,9 @@ jobs:
     permissions:
       id-token: write
       packages: write
+      actions: read
+      contents: write
+      attestations: write
```

Comment:
Not sure `attestations: write` is required within this context.

Reply:
Indeed, it was not needed. Removed now.